### PR TITLE
Update fixtures generation

### LINKED ISSUE #76: Update fixtures generator
**Is your feature request related to a problem? Please describe.**
Since we work on the project, our database model has evolved, and our fixtures generator is not accurate anymore

**Describe the solution you'd like**
Rework on the fixtures generator function

### PR DESCRIPTION
**Pull request by EmileCalixte**
| Q             | A
| ------------- | ---
| Is bugfix?    | Yes and No, it's an upgrade after our development and our database model editions
| New feature?  | No
| Fixed issues  | Fixes #76

### CODE REVIEW

**Review by GuenoleCHERUBIN — COMMENTED** [review]
Missing some PHPDoc

**Comment by EmileCalixte** [comment]
You're right !

File: commands/FixturesController.php [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+            $reviewMinDate = Util::getDateTimeFromDatabaseString($proposal->date);
+            $reviewMaxDate = new \DateTime();
+
+            $reviewsCount = random_int(self::MIN_REVIEWS_PER_PROPOSAL, self::MAX_REVIEWS_PER_PROPOSAL);
+
+            $this->println("Generating $reviewsCount reviews for proposal {$proposal->id} of {$proposal->submitter->email}");
+
+            $reviewersWhoCanReview = $this->reviewers;
+            for($i = 0; $i < count($reviewersWhoCanReview); ++$i) {
+                if($reviewersWhoCanReview[$i]->id === $proposal->id) {
```
Comment (alexisljn) [inline commands/FixturesController.php]: It should be proposal->submitter_id no ?

File: commands/FixturesController.php [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+                $usersWhoCanComment[] = $proposal->submitter;
+            }
+
+            for($i = 0; $i < count($usersWhoCanComment); ++$i) {
+                $commentAuthorIndex = random_int(0, count($usersWhoCanComment)-1);
+                $commentAuthor = $usersWhoCanComment[$commentAuthorIndex];
+
+                $comment = new Comment();
+                $comment->proposal_id = $proposal->id;
+//                var_dump($commentAuthor);
```
Comment (alexisljn) [inline commands/FixturesController.php]: pls remove it

**Review by alexisljn — CHANGES_REQUESTED** [review]

File: commands/FixturesController.php (hunk 1 above)
Comment (EmileCalixte) [reply commands/FixturesController.php]: Of course, my bad

File: commands/FixturesController.php [hunk 3]
```diff
@@ -0,0 +1,607 @@
...
+                    $comment->edited_date = Util::getDateTimeFormattedForDatabase(
+                        \DateTime::createFromFormat('U', random_int(
+                            Util::getDateTimeFromDatabaseString($comment->date)->format('U'),
+                            $commentMaxDate->format('U'),
+                        ))
+                    );
+                }
+
+                if(!$comment->save()) {
+                    var_dump($comment->errors);
```
Comment (alexisljn) [inline commands/FixturesController.php]: remove it pls

**Review by alexisljn — CHANGES_REQUESTED** [review]

**Review by alexisljn — APPROVED** [review]
Solid work ! Congrats !!

**Pull request merged**